ccd-user-profile-importer/values.yaml

```diff
@@ -0,0 +1,25 @@
+ccd:
+  postgresql:
+    enabled: false
+image: 'hmctspublic.azurecr.io/ccd/user-profile-importer:latest'
+memoryRequests: '512Mi'
+cpuRequests: '250m'
+memoryLimits: '2048Mi'
+cpuLimits: '1500m'
+userProfileUrl: http://ccd-user-profile-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/users
+s2sUrl: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+users: 
+jurisdictions:
+waitHosts: ""
+waitHostsTimeout: 300
+microservice: ccd_definition
+verbose: "false"
+userProfileDatabaseHost: "{{ .Release.Name }}-postgresql"
+userProfileDatabasePort: "5432"
```

Comment:
Actually POST "/user-profile/users" endpoint creates jurisdiction record if it's not there already. So, you don't need all these database config, create-jurisdiction.sh etc.

Reply:
it doesn't work, without this configuration, see the way cmc uses this importer
https://github.com/hmcts/chart-cmc/blob/master/cmc/values.yaml#L45

Reply:
They need to remove that unnecessary config once upgrade to latest ccd chart.

Reply:
removed db attributes